sorts/test/sleep_sort.test.ts

```diff
@@ -0,0 +1,39 @@
+import { sleepSort } from "../sleep_sort";
+
```

Comment:
Put the tests in a `describe("Sleep sort", ...)`.

Reply:
Done with the describe. @appgurueu